openedx/adg/common/lib/mandrill_client/client.py

```diff
@@ -65,6 +70,22 @@ def send_mandrill_email(self, template, email, context):
             None
         """
         log.info(f'Sending email using template: {template}, account: {email} and context: {context} using mandrill')
-
+        template = self.add_language_in_template_slug(template)
         email_data = EmailData(template, email, context)
         self._send_mail(email_data)
+
+    def add_language_in_template_slug(self, template):
```

Comment:
Can we rename this to something like `add_user_preferred_language_to_template_slug` for better clarity?